### DISCUSSION TITLE
Multiple contexts

### DISCUSSION THREAD

**Discussion by jamierobertson1** [ocornut/imgui#6734] [New Users Build/Link/Run issues ONLY!]
I'm trying use Imgui (verison 1.89.8) in multiple windows managed by a custom vulkan backend. 

Im creating seperate Imgui contexts for each window and doing ImGui::SetCurrentContext before forwading events to Imgui and also before ImGui::NewFrame()

This partially works, the ui is rendered in each window, but they are unresponsive when more than one window is visible simultaneously (although they repond ok to resize events). If I minimize all except one then that one visible will work as expected.

I've checked that the events are filtered properly for the context and that they are correctly passed to ImGui (even for the unresponsive windows).

Unfortunately it's not particularly easy for me to create a reproduceable example, but just wondering if anyone else has done this successfully or had any pointers?

**Comment by ocornut** [comment]
We are asking this kind of post to be made in Issues. 

Without more details and less ambiguities (which are often lifted by reading issues guidelines) it’s going to be difficult to provide help.
